administrator and tags fixes

awelzel opened the pull request:
Hello,

second attempt. The commit messages should contain enough info.

provider rabbitmq_user
* do not expose the administrator tag

type rabbitmq_user
* do not allow explicit setting of the administrator tag
* default to an empty array of tags
* sort tags before comparison

Let me know if you have any comment.

Thanks,
   Arne

igalic commented:
@awelzel when you add your [email to github](https://help.github.com/articles/setting-your-email-in-git/) your commits will be associated with your account.
also, i'd [squash](http://www.git-scm.com/book/en/Git-Tools-Rewriting-History#Squashing-Commits) those two commits.

awelzel commented:
@igalic , thanks for your input. I've added my other email to github, squashed the commits, and repushed the same branch. I've also added a short description of why I changed things to begin with.

Thanks,
   Arne

nibalizer commented:
lgtm, but i'm not t/p expert

cmurphy commented:
Thanks!
